test/paper_fab_basic_test.dart

```diff
@@ -0,0 +1,53 @@
+@TestOn('browser')
+library polymer_elements.test.paper_fab_basic_test;
+
+import 'dart:html';
+import 'package:polymer/polymer.dart';
+import 'package:polymer_elements/paper_fab.dart';
+import 'package:polymer_elements/iron_icon.dart';
+import 'package:polymer_elements/iron_icons.dart';
+import 'package:test/test.dart';
+import 'package:web_components/web_components.dart';
+import 'common.dart';
+
+Map centerOf(Element element) {
```

Comment:
There is `middleOfNode` in `common.dart` already. It also returns an actual `Point` class which is kind of nice.